iina/PreferenceWindowController.swift

```diff
@@ -350,6 +353,17 @@ class PreferenceWindowController: NSWindowController {
     return nil
   }
 
+  func performAction(_ action: Action) {
+    switch action {
+    case .installPlugin(url: let url):
+      guard let idx = viewControllers.firstIndex(where: { $0 is PrefPluginViewController }) else {
+        return
+      }
+      loadTab(at: idx)
+      let vc = viewControllers[idx] as! PrefPluginViewController
+      vc.perform(#selector(vc.installPluginAction(localPackageURL:)), with: url, afterDelay: 0.25)
```

Comment:
I assume this delay is to make the preference window appear? Is there really no better way to do this?